templates/kubernetes/deploy/deployment.yml

```diff
@@ -23,6 +23,14 @@ spec:
           ports:
             - containerPort: <% index .Params `containerPort` %>
               name: http
+          # Uncomment the env block below if your Playground application has secrets
+          # See the README for instructions on how to create secrets
+          #env:
+          #  - name: TEST_SECRET
+          #    valueFrom:
+          #      secretKeyRef:
+          #        name: <% .Name %>-secrets
+          #        key: test-secret-key
           envFrom:
           - configMapRef:
               name: <% .Name %>-config
```

Comment:
After another thought, I think this might be a slightly easier way for users to use the secret
Kubernetes allows us to reference the entire secret so it will just pick up all the key-value pairs, so they dont have to add the key each time. 
```suggestion
              name: <% .Name %>-config
#             - secretRef:
#              name: app-secrets
#    volumes:
#       - name: app-secrets
#         secret:
#           secretName: <% .Name %>-secrets
```

Reply:
Oh cool! I'll give that a quick try.

Reply:
Fixed

Reply:
@davidcheung Doesn't look like a volume is required here, we can just reference the secrets file directly in the `envFrom` block to load in all the secrets.